api/routers/edr.py

```diff
@@ -255,10 +261,9 @@ async def get_data_area(
         )
 
     range = get_datetime_range(datetime)
-    # await verify_parameter_names(parameter_name)
     if parameter_name:
-        parameter_name = parameter_name.split(",")
-        parameter_name = list(map(lambda x: x.strip(), parameter_name))
+        parameter_name = split_and_strip(parameter_name)
+        await verify_parameter_names(parameter_name)
     request = dstore.GetObsRequest(
         filter=dict(parameter_name=dstore.Strings(values=parameter_name if parameter_name else None)),
```

Comment:
Should set `parameter_name` to None in the dict if no parameters are given, not but None in the `Strings` like we do now.

Reply:
Will make separate issue for this.